linode/acceptance/util.go

```diff
@@ -689,3 +689,62 @@ func GetTestClient() (*linodego.Client, error) {
 
 	return client, nil
 }
+
+var SmokeTests = map[string][]string{
```

Comment:
This map variable contains all the smoke test cases across test packages in each directory. We will need to add new smoke tests to this map